opensearch/descriptions/datasets.xml

```diff
@@ -13,7 +13,7 @@
   DCO authors of the datasets, non-DCO authors of the datasets, and the
   projects the datasets are related to), and the result set.</Description>
   <Tags></Tags>
-  <Contact>web@mail.deepcarbon.net</Contact>
+  <Contact>Han Wang</Contact>
```

Comment:
We don't want to make this change.  Ditto for some of the other opensearch description document changes.